cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -175,9 +175,9 @@ def single_qubit_randomized_benchmarking(
     qubit: 'cirq.Qid',
     use_xy_basis: bool = True,
     *,
-    num_clifford_range: Sequence[int] = range(10, 100, 10),
-    num_circuits: int = 20,
-    repetitions: int = 1000,
+    num_clifford_range: Sequence[int] = tuple(np.logspace(np.log10(5), 3, 5, dtype=int)),
+    num_circuits: int = 10,
+    repetitions: int = 600,
```

Comment:
Why this change to repetitions? I suspect the runtime for 600 vs 1000 repetitions will be almost indistinguishable since the runtime is likely not dominated by hardware reps.